### DISCUSSION TITLE
[Beginner] Finding Help

### DISCUSSION THREAD

**Discussion by Kha1r0** [ianyh/Amethyst#1688] [Q&A]
Hey There!
I've read the manual and looked up a few tutorials and tried around for a few hours but i'm still struggling with two aspects and i'm wondering where i can find help. Is there a discord server for that or am i already in the right place to ask my questions?

1. i cannot figure out how to get 4 windows with each one filling a quarter of the screen to work. Is that only possible with the Binary Space Partitioning Layout?

2. i have played around with binary space partitioning window managers in the past but thats not at all how the binary space partitioning layout behaves on my machine to the point where i think it may just be completely buggy/broken on my machine. 
Some things ive noticed are:

2.2 sometimes, the first window thats opened defaults to one of the screen halves even though it's the only open window, leaving the other screen half blank.

2.1 When i add more windows to one existing window, it either doesnt tile at all and just floats in front (if i press opt+shift+ctrl+T, nothing changes, if i drag it around with the mouse, it doesnt tile in any way), or it doesn't halve the window i had focused before.

2.3 imagine i have one big window on the left, and two smaller windows on the right half of the screen. How do i, for example, tell the lower right window to instead share it's screen space on the left with the left window, making the top right window big? Whats the intended workflow for that?

Thank you for reading. I'm sorry if i've made some kind of silly mistake.

**Comment by Kha1r0** [comment]
To improve upon my description of problems with BSP, it seems like BSP is seeing some kind of another window, but there's nothing there.
I have also been able to replicate this problem in a VM.
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/32c5ba0c-0194-4e3c-8fc9-560be0bf5602">

**Reply by rb-411** [reply]
Mine is doing the same thing and it's really frustrating. Have you found a solution?
